Chaincase/ViewModels/ReceiveViewModel.cs

```diff
@@ -39,7 +39,7 @@ public ReceiveViewModel(IScreen hostScreen) : base(hostScreen)
 
 			GenerateCommand = new Command(() =>
 			{
-				Label = Label.Trim(',', ' ').Trim();
+				Label = Label == null ? "" : Label.Trim(',', ' ').Trim();
```

Comment:
I think I led you astray on this -- Every address must be labeled or else we can't keep track of the anonymity of the utxo

Reply:
My cheap UX preserving solution to this is to type (required -- will crash otherwise) in the field Label for now

Reply:
I fixed most of the requested changes, please ensure that is accurate